### ISSUE TITLE
Improve jobsize estimate when user specifies 'events per job' on collisions data

### ISSUE THREAD

**Issue by annawoodard** [matz-e/lobster#110]
When using the DBS interface, if the user specifies 'events per job', we estimate the jobsize as ```jobsize = jobits / job = events / job / (average events / lumisection)```. For collisions data, it would be preferable to take ```average events / lumisection``` from ```file events / file lumisections``` for the corresponding file (and let the jobsize change depending on the file) rather than ```total events / total lumisections``` as we do [currently](https://github.com/matz-e/lobster/blob/master/lobster/cmssw/dataset.py#L52).

**Comment by matz-e** [comment]
Agreed, although I think this should be a property of the lumi-section / jobit, to allow greater abstraction of the job creation, as in #65.

**Comment by annawoodard** [comment]
That reminds me: we'll have to modify somewhat our current approach to grabbing jobits from the DB, because we'll need to ensure that all jobits in the job have the same jobsize.

**Comment by matz-e** [comment]
IMO another reason to save the info in the jobit table?

**Comment by annawoodard** [comment]
OK. So, drawing from the discussion in #108, I think we're starting to converge here. Drop the current 'file', 'lumi', and 'run' columns for one 'id' column with a tuple whose structure depends on the jobit unit (we already use id for the row number, so I'll call it 'specification'):

| jobit unit | jobit specification |
|:---|:---|
| lumi | (file, run, lumi, jobsize, args) |
| file | ((file, jobsize, args )|
| event | (file, skip_events, max_events, jobsize, args) |

Does that look right?

**Comment by matz-e** [comment]
`jobsize` should still be a per-dataset property?
